### ISSUE TITLE
One more comparison for the record.

### ISSUE THREAD

**Issue by dblodgett-usgs** [USGS-R/EflowStats#132]
I ran across some old comparison records from Jessica. Worked them up into an example. Will PR the data and script that produced this too. Note that this comparison is between the new R code and the old pre-R code.
```
    statistic new_result old_result abs_diff    perc_diff
1         ma1    198.686    198.686    0.000   0.00000000
2         ma2    161.000    161.000    0.000   0.00000000
3         ma3     77.225     77.690    0.465   0.60032921 Known discrepincies in old code. #82
4         ma4     11.084     55.452   44.368 133.36539618 Noted issues with not using log of flows. #122
5         ma5      1.234      1.234    0.000   0.00000000
6         ma6      5.014      5.014    0.000   0.00000000
7         ma7      2.781      2.781    0.000   0.00000000
8         ma8      2.250      2.250    0.000   0.00000000
9         ma9      0.317      1.720    1.403 137.75159548 Noted issues with not using log of flows. #122
10       ma10      0.201      1.062    0.861 136.34204276 Noted issues with not using log of flows. #122
11       ma11      0.160      0.839    0.679 135.93593594 Noted issues with not using log of flows. #122
12       ma12    276.444    276.444    0.000   0.00000000
13       ma13    282.047    282.139    0.092   0.03261336
14       ma14    311.695    311.695    0.000   0.00000000
15       ma15    270.715    270.715    0.000   0.00000000
16       ma16    209.276    209.276    0.000   0.00000000
17       ma17    172.085    172.085    0.000   0.00000000
18       ma18    130.258    130.258    0.000   0.00000000
19       ma19    140.498    140.498    0.000   0.00000000
20       ma20    113.574    113.574    0.000   0.00000000
21       ma21    129.297    129.297    0.000   0.00000000
22       ma22    157.567    157.567    0.000   0.00000000
23       ma23    195.642    195.642    0.000   0.00000000
24       ma24     57.532     57.532    0.000   0.00000000
25       ma25     48.790     48.790    0.000   0.00000000
26       ma26     43.962     43.962    0.000   0.00000000
27       ma27     28.929     28.929    0.000   0.00000000
28       ma28     30.695     30.695    0.000   0.00000000
29       ma29     34.228     34.228    0.000   0.00000000
30       ma30     39.391     39.391    0.000   0.00000000
31       ma31     48.455     48.455    0.000   0.00000000
32       ma32     45.464     45.464    0.000   0.00000000
33       ma33     60.954     60.954    0.000   0.00000000
34       ma34     51.337     51.337    0.000   0.00000000
35       ma35     57.202     57.202    0.000   0.00000000
36       ma36      2.022      2.022    0.000   0.00000000
37       ma37      0.738      0.738    0.000   0.00000000
38       ma38      1.454      1.454    0.000   0.00000000
39       ma39     48.930     48.930    0.000   0.00000000
40       ma40      0.067      0.067    0.000   0.00000000
41       ma41      3.517      3.517    0.000   0.00000000
42       ma42      0.470      0.470    0.000   0.00000000
43       ma43      0.178      0.178    0.000   0.00000000
44       ma44      0.470      0.470    0.000   0.00000000
45       ma45     -0.031     -0.031    0.000   0.00000000
46        ml1    143.778    143.778    0.000   0.00000000
47        ml2    176.111    176.111    0.000   0.00000000
48        ml3    198.111    198.111    0.000   0.00000000
49        ml4    194.111    194.111    0.000   0.00000000
50        ml5    140.444    140.444    0.000   0.00000000
51        ml6    112.000    112.000    0.000   0.00000000
52        ml7     85.556     85.556    0.000   0.00000000
53        ml8     76.556     76.556    0.000   0.00000000
54        ml9     75.778     75.778    0.000   0.00000000
55       ml10     72.444     72.444    0.000   0.00000000
56       ml11     84.667     84.667    0.000   0.00000000
57       ml12    112.667    112.667    0.000   0.00000000
58       ml13     47.402     47.402    0.000   0.00000000
59       ml14      0.302      0.302    0.000   0.00000000
60       ml15      0.250      0.250    0.000   0.00000000
61       ml16      0.291      0.291    0.000   0.00000000
62       ml17      0.277      0.311    0.034  11.56462585 Scrutinized Code, error in old code, see #136
63       ml18     28.817     34.163    5.346  16.97681804 Scrutinized Code, error in old code, see #136
64       ml19     25.043     25.043    0.000   0.00000000
65       ml20      0.704      0.704    0.000   0.00000000
66       ml21     23.991     23.991    0.000   0.00000000
67       ml22      0.871      0.871    0.000   0.00000000
68        mh1    917.333    917.333    0.000   0.00000000
69        mh2    826.444    826.444    0.000   0.00000000
70        mh3    871.778    871.778    0.000   0.00000000
71        mh4    566.111    566.111    0.000   0.00000000
72        mh5    439.222    439.222    0.000   0.00000000
73        mh6    376.111    376.111    0.000   0.00000000
74        mh7    378.000    378.000    0.000   0.00000000
75        mh8    540.778    540.778    0.000   0.00000000
76        mh9    334.556    334.556    0.000   0.00000000
77       mh10    543.333    543.333    0.000   0.00000000
78       mh11    429.778    429.778    0.000   0.00000000
79       mh12    691.444    691.444    0.000   0.00000000
80       mh13     77.525     77.525    0.000   0.00000000
81       mh14      8.691      8.691    0.000   0.00000000
82       mh15      5.109      5.109    0.000   0.00000000
83       mh16      2.149      2.149    0.000   0.00000000
84       mh17      1.509      1.509    0.000   0.00000000
85       mh18      4.596      4.596    0.000   0.00000000
86       mh19     -0.558     -0.558    0.000   0.00000000
87       mh20     27.133     27.133    0.000   0.00000000
88       mh21     10.243     10.166    0.077   0.75456906
89       mh22      3.152      3.152    0.000   0.00000000
90       mh23      2.719      2.719    0.000   0.00000000
91       mh24      2.584      2.576    0.008   0.31007752
92       mh25      5.208      5.208    0.000   0.00000000
93       mh26      9.366      9.366    0.000   0.00000000
94       mh27      3.384      3.370    0.014   0.41456914
95        fl1      9.667      9.667    0.000   0.00000000
96        fl2     42.021     42.021    0.000   0.00000000
97        fl3      0.000      0.000    0.000          NaN Scruitinized in #20
98        fh1     14.667     14.667    0.000   0.00000000 fh stats scruitinized in #69
99        fh2     27.059     27.059    0.000   0.00000000
100       fh3     15.222     15.222    0.000   0.00000000
101       fh4      1.778      1.778    0.000   0.00000000
102       fh5     14.778     14.778    0.000   0.00000000
103       fh6      7.778      7.778    0.000   0.00000000
104       fh7      1.556      1.556    0.000   0.00000000
105       fh8     14.667     14.667    0.000   0.00000000
106       fh9      9.222      9.222    0.000   0.00000000
107      fh10      2.889      2.889    0.000   0.00000000
108      fh11         NA      1.444       NA           NA
109       dl1     49.222     49.222    0.000   0.00000000 All dl stats scruitinized in #61, #67 
110       dl2     52.095     50.778    1.317   2.56043860 
111       dl3     57.367     55.222    2.145   3.81031895 
112       dl4     64.633     66.248    1.615   2.46789068 
113       dl5     81.906     99.140   17.234  19.03825547 
114       dl6     23.991     23.991    0.000   0.00000000
115       dl7     22.885     26.428    3.543  14.36943605 
116       dl8     30.057     30.663    0.606   1.99604743 
117       dl9     32.662     32.773    0.111   0.33926798 
118      dl10     34.140     29.326    4.814  15.17032742 
119      dl11      0.306      0.306    0.000   0.00000000
120      dl12      0.356      0.343    0.013   3.71959943 
121      dl13      0.401      0.411    0.010   2.46305419 
122      dl14      0.671      0.671    0.000   0.00000000
123      dl15      0.429      0.429    0.000   0.00000000
124      dl16      7.222      7.222    0.000   0.00000000
125      dl17     76.806     76.806    0.000   0.00000000
126      dl18      0.000      0.000    0.000          NaN
127      dl19        NaN         NA      NaN          NaN
128      dl20      0.000      0.000    0.000          NaN
129       dh1   1533.000   1533.000    0.000   0.00000000 All dh stats scruitinized in #61, #67 
130       dh2    980.407    980.407    0.000   0.00000000
131       dh3    642.000    644.058    2.058   0.32004777 
132       dh4    416.230    416.597    0.367   0.08813355
133       dh5    323.988    324.076    0.088   0.02715781
134       dh6     31.038     31.038    0.000   0.00000000
135       dh7     35.425     35.425    0.000   0.00000000
136       dh8     27.529     27.260    0.269   0.98194893
137       dh9     20.313     20.402    0.089   0.43718531
138      dh10     19.904     20.206    0.302   1.50585889
139      dh11      9.522      9.522    0.000   0.00000000
140      dh12      3.988      4.000    0.012   0.30045068
141      dh13      2.585      2.588    0.003   0.11598685
142      dh14      1.936      1.936    0.000   0.00000000
143      dh15      5.750      5.750    0.000   0.00000000
144      dh16     62.001     62.001    0.000   0.00000000
145      dh17     12.409     16.084    3.675  25.79580950
146      dh18      1.957      1.945    0.012   0.61506920
147      dh19      1.143      1.119    0.024   2.12201592
148      dh20      6.260      6.766    0.506   7.76907723
149      dh21     30.600     37.842    7.242  21.16244411
150      dh22         NA     77.487       NA           NA
151      dh23         NA      2.000       NA           NA
152      dh24         NA    143.111       NA           NA
153       ta1      0.626      0.626    0.000   0.00000000 TA stats scruitinized in #74
154       ta2     75.216     75.201    0.015   0.01994455
155       ta3        NaN      0.385      NaN          NaN
156       tl1      0.487    274.016  273.529 199.29035384
157       tl2     28.097     27.709    0.388   1.39053148
158       tl3         NA      0.667       NA           NA
159       tl4         NA      0.011       NA           NA
160       th1     76.005    348.880  272.875 128.44652082
161       th2     64.846     64.883    0.037   0.05704199
162       th3         NA      0.315       NA           NA
163       ra1     82.245     82.245    0.000   0.00000000
164       ra2    213.764    213.764    0.000   0.00000000
165       ra3    -29.905     29.905   59.810          Inf
166       ra4   -272.893    272.893  545.786          Inf
167       ra5      0.259      0.259    0.000   0.00000000
168       ra6      0.151      0.151    0.000   0.00000000
169       ra7     -0.061      0.061    0.122          Inf
170       ra8    116.222    115.222    1.000   0.86413992
171       ra9      5.404      5.451    0.047   0.86596039
```

**Comment by dblodgett-usgs** [comment]
@tmills-usgs you are more familiar with the implementation of the bottom third of these stats that have small differences. Can you comment them up and see if there are any that we need to scrutinize more closely?

**Comment by dblodgett-usgs** [comment]
I just went through all the closed tickets and put issue numbers into the comments of the doc above. I opened one new issue, otherwise, I feel pretty confident that these have been scrutinized enough.

**Comment by dblodgett-usgs** [comment]
I think we can call this closed and move on to #126
